Sources/Defaults/Defaults.swift

```diff
@@ -292,13 +292,64 @@ extension Defaults {
 		}
 	}
 
-	// We still keep this as it can be useful to pass a dynamic array of keys.
 	/**
 	Observe updates to multiple stored values.
 
 	- Parameter keys: The keys to observe updates from.
 	- Parameter initial: Trigger an initial event on creation. This can be useful for setting default values on controls.
 
+	```swift
+	Task {
+		for await (foo, bar) in Defaults.updates([.foo, .bar]) {
+			print("Values changed:", foo, bar)
+		}
+	}
+	```
+	*/
+	public static func updates<each Value: Serializable>(
+		_ keys: repeat Key<each Value>,
+		initial: Bool = true
+	) -> AsyncStream<(repeat each Value)> {
+		.init { continuation in
+			func getCurrentValues() -> (repeat each Value) {
+				(repeat self[each keys])
+			}
+
+			var observations = [DefaultsObservation]()
+
+			if initial {
+				continuation.yield(getCurrentValues())
+			}
+
+			for key in repeat (each keys) {
+				let observation = DefaultsObservation(object: key.suite, key: key.name) { _, _  in
```

Comment:
Note: I'm intentionally not using CompositeDefaultsObservation for simplicity, as it's not really needed here.